modules/global/src/com/haulmont/cuba/core/global/View.java

```diff
@@ -140,11 +145,16 @@ public View(ViewParams viewParams) {
                 addProperty(propertyName);
             }
         }
-        if (viewParams.src != null) {
-            this.properties.putAll(viewParams.src.properties);
-            if (entityClass == null) {
-                this.entityClass = viewParams.src.entityClass;
+        List<View> sources = viewParams.src;
+
+        if (sources != null && !sources.isEmpty()) {
+            Class<? extends Entity> entityClass = sources.get(0).entityClass;
+
+            if (this.entityClass == null) {
+                this.entityClass = entityClass;
             }
+
+            sources.forEach(s -> this.properties.putAll(s.properties));

```

Comment:
What if we have clashed properties in the sources views with different views? Are they simply ignored?